### DISCUSSION TITLE
How can I find a receipt / order's refund amount in my own currency that the sale was made in?

### DISCUSSION THREAD

**Discussion by casualbobuk** [etsy/open-api#813] [Q&A]
Hi, my shop sells in GBP. 

If someone buys in America in USD and I refund them, the Etsy website lists the refund in GBP (I'm not shown dollar amounts anywhere, it does it all for me), but the API serves the refund amount in USD, seemingly without any further info to convert this into GBP.

How can I either access the GBP value, or find out what exchange rate Etsy used at the time of the refund via the API please? I currently can't programatically consolidate my ins and outs.

Many thanks.

Dave

**Comment by daliuss** [comment]
+1
This is frustrating. I cannot find the refund amount in the original currency for some of the orders.

**Comment by SimasBir** [comment]
+1 as well.
Current refund amounts don't match totals with the unknown exchange rate and possibly the additional etsy conversion fees.
For example, a sale with a grandtotal 120.07 EUR, Etsy refunded cost shows 120.07 EUR whereas API has a refund with the amount 135.69 USD (122-124 EUR with current exchange rates).
Would like to see the refund in the sellers currency currency.

**Comment by Gareth064** [comment]
@casualbobuk  @daliuss  @SimasBir one of you should email developer@etsy.com and ask them to look into this for you. Either its a bug or a feature not implemented yet. If you do email them, mention this GitHub discussion and then report back here what they respond with.

**Reply by SimasBir** [reply]
Will do, thanks for the reply
